en/firefox/browsers/mobile/compare.ftl

```diff
@@ -0,0 +1,64 @@
+
+mobile-compare-which-mobile-browser = Which mobile browser keeps things confidential?
+mobile-compare-at-a-minimum = At a minimum, your mobile browser should provide some version of “private browsing mode,” which automatically deletes your history and search history. In this area, all seven of the browsers compared here score points.
+mobile-compare-another-mobile-feature = Another mobile feature that should be a given is the ability to prevent websites and companies from tracking your browsing and shopping data — even in normal browsing mode.
+mobile-compare-blocking-thirdparty-trackers = Blocking third-party trackers isn’t just important for privacy — it also helps pages load much faster, without those pieces of code attaching themselves and slowing your browser down.
+mobile-compare-from-the-chart = From the chart above, it’s obvious { -brand-name-firefox } offers the most complete privacy and security toolset of the five browsers and provides a dashboard overview of your overall security and privacy online.
+
+# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. If the translation requires multiple words, please choose a word to emphasize and wrap that word in the strong tag.
+mobile-compare-features-strong = <strong>Features</strong>
+mobile-compare-features = Features
+mobile-compare-what-can-your = What can your browser do for you?
+mobile-compare-a-main-ingredient = A main ingredient to a well-made browser is its functionality. What can it do?  Almost all five browsers are equal when it comes to tabs, bookmarks, auto-completion of fields, but only { -brand-name-firefox } and { -brand-name-safari } offer add-ons/extensions, which are kind of like apps for your browser to make it more private, powerful or just more fun.
+
+# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. If the translation requires multiple words, please choose a word to emphasize and wrap that word in the strong tag.
+mobile-compare-syncing-strong = <strong>Syncing</strong>
+mobile-compare-syncing = Syncing
+mobile-compare-does-your-browser = Does your browser play well with other devices?
+mobile-compare-the-mobile-version = The mobile version of { -brand-name-safari }, which is pre-installed on { -brand-name-apple }’s mobile devices, is only compatible with other { -brand-name-apple } devices like { -brand-name-ipad } and laptops. { -brand-name-firefox }, { -brand-name-chrome }, and { -brand-name-opera } work across all platforms including { -brand-name-ios }, { -brand-name-android }, { -brand-name-windows }, { -brand-name-mac-short }, and { -brand-name-linux }, with the exception of the { -brand-name-edge } browser which isn’t compatible with { -brand-name-linux }-based devices.
```

Comment:
Should we mention iPhone here?

Reply:
Maybe iPhone is implied, but I do think we should specify "Mac laptops". I'll add that.